packages/xo-web/src/common/xo/index.js

```diff
@@ -3796,3 +3798,29 @@ const _callGithubApi = async (endpoint = '') => {
 export const getMasterCommit = () => _callGithubApi('/commits/master')
 
 export const compareCommits = (base, head) => _callGithubApi(`/compare/${base}...${head}`)
+
+export const uuidToLink = text => {
+  const regex = /\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b/g
```

Comment:
Are the `\b`s in the middle necessary? Isn't this enough?
`/\b[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\b/g`